<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebitCalc</class>
 <widget class="QWidget" name="DebitCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>769</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Сумма вклада</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="date_edit_begin"/>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>% годовых</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="credit_debit_combo_box">
     <item>
      <property name="text">
       <string>Пополнение</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Снятие</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Срок размещения</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="kap_check_box">
     <property name="text">
      <string>Капитализация процентов</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="line_per">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Начало срока</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Процентная ставка</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="period_combo_box">
     <item>
      <property name="text">
       <string>раз в месяц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>раз в год</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>в конце срока</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>     Пополнения:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="line_sum">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Периодичность выплат</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="line_time">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Снятия:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="date_combo_box">
     <item>
      <property name="text">
       <string>Месяцев</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Лет</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDateEdit" name="date_edit_add"/>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="line_add"/>
   </item>
   <item row="9" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableView" name="debit_table_view">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="credit_table_view">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="add_btn">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="del_button">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="calc_btn">
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QTableView" name="final_table_view"/>
   </item>
   <item row="11" column="3" colspan="2">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Проценты</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lab_proc">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lab_tax">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Сумма</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lab_sum">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Налог</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Ключевая ставка ЦБ</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="line_key_rate"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
